ENG4000-Team-A/capstone-project
# Machine overview and

**New Features:**
All the pages are created and connected
- #52
- #53 
- #21

Countdown and timer page are based on each machine. Connected the model and the view so that time left on the countdown page persists when the page is reloaded.
- #60

**Restrictions:**
- I set a default user (User with id 1)
- I made the user only be able to set the remaining time on their account to the machine.
- User can use multiple machines.

**Instructions on setup and testing:**
**setup:**
1. Run: python3 manage.py runserver
2. Go to http://127.0.0.1:8000/admin
3. Create as many Machines as wanted, add at least one user (We only use one for now) make sure the time on the User > 0 make around 30 seconds to be able to quickly test everything

**testing:** 
1. Go to http://127.0.0.1:8000/machines
   - All the machines you selected should appear
2. Select a machine
   - you should be redirected to the set timer page
   - the time you set for the first user should show at the top
3. Click the submit button
   - you should be redirected to the countdown page the time should start from the time you selected 
5. Try reloading the page 
   - The timer should still be accurate
6. Try visiting other pages to see if everything makes sense
7.  When the timer ends the time should be reflected in the user model.
